docs/development/setup_development.rst

```diff
@@ -164,15 +164,6 @@ from the `Vagrant Downloads page`_ and then install it.
             instructions in Vagrantfile that would enable vagrant-cachier are
             currently commented out.
 
-.. todo:: This warning is here because a common refrain during hackathons for
```

Comment:
irrelevant, we use the Docker dev env at hackathons (rarely if ever to people wade into the waters :ocean: of the staging VMs at hackathons)